src/components/core/Input/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classes from './Input.module.scss';
+
+interface InputProps {
+  type: string;
+  placeholder: string;
+  id: string;
+  name: string;
+  label: string;
+  setValue: (value: string) => void;
+}
+
+const Input: React.FC<InputProps> = ({ type, placeholder, id, name, label, setValue }) => {
+  return (
+    <div className={classes['c-input']}>
+      <label className={classes['c-input__label']} htmlFor={id}>
+        {label}
+      </label>
```

Comment:
Since I suggested changing label to be optional, conditional rendering should be used on line 16.